docs/book/features/block_volume.md

```diff
@@ -376,6 +378,47 @@ cluster. Make sure to mention `pv.kubernetes.io/provisioned-by: csi.vsphere.vmwa
         Events:                <none>
     ```
 
+### Statically Provision a Block Volume in vSphere with Kubernetes<a id="static_volume_provisioning_vsphere_with_k8s"></a>
+
+vSphere 7.0 does not support static volume provisioning in vSphere with Kubernetes aka supervisor cluster. Static volume provisioning is a process where an existing volume on the storage is made available in the Kubernetes cluster. There are use cases where we want to import existing volumes into supervisor cluster.
+
+A new CnsRegisterVolume API is introduced in vSphere with Kubernetes which will allow to import a volume into supervisor namespace. CnsRegisterVolume API objects are namespaced objects and once created successfully it would create a PVC in that namespace and a PV bound to that PVC.
```

Comment:
Can you mention the vSphere release where this API is introduced? Maybe something like this:

A new CnsRegisterVolume API is introduced in vSphere 7.0 Update1 release in vSphere with Kubernetes that allows importing an existing CNS volume or FCD or vmdk into a supervisor namespace.